Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners.py

```diff
@@ -119,11 +117,14 @@ def aggregate(owners: List[Dict[str, str]]) -> List[Dict[str, Any]]:
         # aggregate remaining keys according to type
         all_keys = set(k for owner in duplicates for k in owner.keys())
         keys_to_types = {k: type(owner[k]) for owner in duplicates for k in owner.keys()}
-        other_keys = all_keys - {'Name', 'Email', 'Source', 'Timestamp', 'Canonicalization'}
+        other_keys = []
+        for key in all_keys:
+            if key.lower() not in {'name', 'email', 'source', 'timestamp', 'canonicalization'}:
+                other_keys.append(key)
         for other in other_keys:
             if keys_to_types[other] == str:
                 # union over strings
-                owner[other] = ' | ' .join(sorted(
+                owner[other] = ',' .join(sorted(
```

Comment:
Can we retain " | " across the board (instead of ",")?

I ask because "," is much more frequent, so it's much more fragile to parse independent sources from each other  downstream. Specifically, the following becomes much more dangerous/no longer reasonable: `owner.get("sources", "").split(",")`.

Reply:
That's fine with me. I think our plan is to make `Justification` be a more human-readable string, so maybe the UI can read from that field instead of `Source`. Alternatively if downstream users want a comma-separated list of sources they can `.replace(' | ', ',')`

@BigEasyJ what do you think?